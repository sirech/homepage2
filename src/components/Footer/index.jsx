import React from 'react'
import cx from 'classnames'

import styles from './style.module.scss'

export default () => (
  <footer className={cx(styles.footer, 'text-center')}>
    <div>
      Copyright Â© {new Date().getFullYear()},{' '}
      <a
        href="http://creativecommons.org/licenses/by/4.0/legalcode"
        target="_blank"
        rel="noopener noreferrer"
      >
        Licensed under CC-by-4.0
      </a>
    </div>
    Favicon courtesy of{' '}
    <a href="https://icons8.com/" target="_blank" rel="noopener noreferrer">
      Icons8
    </a>{' '}
  </footer>
)
