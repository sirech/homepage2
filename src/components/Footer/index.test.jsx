import React from 'react'
import { shallow } from 'enzyme'
import Footer from './index'

describe('components', () => {
  describe('Footer', () => {
    it('renders correctly', () => {
      const component = shallow(<Footer />)
      expect(component).toHaveLength(1)
    })
  })
})
