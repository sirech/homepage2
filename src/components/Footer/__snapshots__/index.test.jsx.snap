// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components Footer renders correctly 1`] = `
<footer
  className="footer text-center"
>
  <div>
    Copyright Â© 
    2020
    ,
     
    <a
      href="http://creativecommons.org/licenses/by/4.0/legalcode"
      rel="noopener noreferrer"
      target="_blank"
    >
      Licensed under CC-by-4.0
    </a>
  </div>
  Favicon courtesy of
   
  <a
    href="https://icons8.com/"
    rel="noopener noreferrer"
    target="_blank"
  >
    Icons8
  </a>
   
</footer>
`;
